aws/sqs/main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  # This helps avoid queue names ending in "-" or "-.fifo"
+  given_queue_name = var.queue_name == "" ? "" : "-${var.queue_name}"
+  # All fifo queues must end in .fifo, per AWS rules
+  queue_suffix    = var.is_fifo == true ? ".fifo" : ""
```

Comment:
```var.is_fifo ? ".fifo" : ""```

Reply:
Oh, we have the `== is true` in a couple of other places, I think? I thought I just needed it. I'll fix that.

Reply:
If you find any others, feel free to lump in a cleanup of 'em 😁